samples/openthread/coap_client_multiprotocol_ble/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+# set(NRF_SUPPORTED_BOARDS
+# 	nrf52840_pca10056
```

Comment:
incorrect board names. The names are different now. Using those still works, but shows a "depricated" warning. In current form this will also prevent using new board names.